src/manage_nsfs/manage_nsfs_validations.js

```diff
@@ -65,6 +66,34 @@ async function validate_input_types(type, action, argv) {
     }
 }
 
+/**
+ * validate_no_leading_space ensures that flags do not contain leading spaces
+ *  - if `argv_` contains more than two elements (type and action), it indicates an invalid flag value.
+ *
+ *
+ * ### CLI Parsing Behavior:
+ * When passing arguments via CLI, if a flag's value starts with a space or consists only of whitespace,
+ * the shell may treat it differently:
+ *  - Leading spaces - might be preserved when quoted (`--new_buckets_path=" abc/"`).
+ *  - Unquoted values with spaces - may cause incorrect parsing (`--new_buckets_path= abc/` is interpreted as an empty value).
+ *  - The parsed argument structure typically follows:
+ *      ```json
+ *      { _: [ "command", "action" ], flag1: "value1", flag2: "value2", ... }
+ *      ```
+ *  - The `_` array should contain only the command and action. If additional elements appear,
+ *    it suggests an issue with flag values (e.g., an incorrectly formatted or empty value).
+ *
+ * @param {string[]} argv_
+ * @throws {ManageCLIError}
+ */
+function validate_no_leading_space(argv_) {
+    // checking if 'argv_' contains more then 2 values (i.e, type and action)
+    if (argv_.length > 2) {
+        const details = `flag value is invalid - leading spaces detected in: ${JSON.stringify(argv_.slice(2))}, ensure flag values are properly formatted.`;
```

Comment:
Why `JSON.stringify(argv_.slice(2)`? (could you add a comment about it?)
Did you want to ignore the `type` and `action`? I would mention that not every command has `action` - for example, `whitelist`.

Reply:
after validating type and action we can remove the first two args - anyway we are printing/returning the error  when args length is more than `2`. So I think the check is good enough.
It would be printed like this below(prints the value only and not the type and action):
`"detail": "flag value is invalid - leading spaces detected in: [\"12345678912345678ABC\"], ensure flag values are properly formatted."`